src/services/cohortCreation/index.ts

```diff
@@ -16,10 +23,25 @@ const getPatients = memoize(
     if (!query) return []
     const response = await apiFhir.get<FHIR_API_Response<IPatient>>(query)
     const patients = getApiResponseResources(response)
-    return patients ? patients.map((patient) => patient.id) : []
+    return patients?.map((patient) => patient.id) ?? []
   }
 )
 
+const getPatientsFromDocuments = memoize(
+  async (query: string, filter: string): Promise<(string | undefined)[]> => {
```

Comment:
can you rename the args to: documentsQuery and patientsFilter ? it's not obvious